main.gms

```diff
@@ -85,9 +85,9 @@
 * 
 * Regionscode: 62eff8f7
 * 
-* Input data revision: 6.298001
+* Input data revision: 6.304
 * 
-* Last modification (input data): Mon Apr 11 14:41:17 2022
+* Last modification (input data): Fri May 20 12:34:46 2022
 * 
```

Comment:
In general, we usually don't modify the run names and settings of the main.gms. You can avoid committing all of a file by using someting like ``git add -pu``. Easiest way to get default settings in main.gms is to run ``Rscript start.R`` and commit the resulting main.gms.